docs/sdk-and-tools/sdk-py/sdk-py-cookbook-v1.md

```diff
@@ -0,0 +1,3360 @@
+---
+id: sdk-py-cookbook
+title: Cookbook (v1)
+pagination_prev: sdk-and-tools/sdk-py/sdk-py-cookbook-v0
+pagination_next: sdk-and-tools/sdk-py/installing-mxpy
+---
+
+[comment]: # (mx-abstract)
+
+[comment]: # (mx-context-auto)
+
+## Overview
+
+This page will guide you through the process of handling common tasks using the MultiversX Python SDK (libraries) **v1 (latest, stable version)**.
+
+:::note
+All examples depicted here are captured in **(interactive) [Jupyter notebooks](https://github.com/multiversx/mx-sdk-py/blob/feat/next/examples/v1.ipynb)**.
+:::
+
+We are going to use the [multiversx-sdk-py](https://github.com/multiversx/mx-sdk-py) package. This package can be installed directly from GitHub or from [**PyPI**](https://pypi.org/project/multiversx-sdk/).
+
+<!-- BEGIN_NOTEBOOK { "url": "https://raw.githubusercontent.com/multiversx/mx-sdk-py/refs/heads/feat/next/examples/v1.ipynb" } -->
```

Comment:
Once the sdk v1 is released, we'll change this to fetch the cookbook from the `main` branch.